src/prebid.js

```diff
@@ -995,21 +998,23 @@ $$PREBID_GLOBAL$$.getHighestCpmBids = function (adUnitCode) {
  * @alias module:pbjs.markWinningBidAsUsed
  */
 $$PREBID_GLOBAL$$.markWinningBidAsUsed = function (markBidRequest) {
```

Comment:
It may be just a stylistic difference, but I think it'd be better to not have this method rather than have it silently do nothing.